device-types/Cisco/N9K-C9372TX-E.yaml

```diff
@@ -0,0 +1,152 @@
+---
+manufacturer: Cisco
+model: Nexus 9372TX-E
+part_number: N9K-C9372TX-E
+slug: n9k-C9372tx-e
```

Comment:
The slug should have `cisco-` prepended to it

Reply:
The slug has been prepended with "cisco-".